src/wasmcloud.rs

```diff
@@ -74,6 +73,10 @@ struct Cli {
     /// Specifies a manifest file to apply to the host once started
     #[structopt(long = "manifest", short = "m", parse(from_os_str))]
     manifest: Option<PathBuf>,
+
+    /// attach a label to the host - can be used multiple times
+    #[structopt(long = "label", short = "l", parse(try_from_str = parse_key_val), number_of_values = 1)]
```

Comment:
If multiple values can be specified, what does `number_of_values` do here? Does it just indicate that I shouldn't be providing labels in the form of:
```
--label foo=bar,foobar=barfoo
```
?
I have tested it and it does work when I specify multiple labels as an argument (e.g. `-l this=that -l  that=this`) but I was just curious.

Reply:
Yep that's it. It restricts each occurrence of the flag to just one pair. Without it, you could do `wasmcloud --label x=1 y=2` but that would prevent you from following it with any other arguments.

Reply:
(although a comma would fix that, I guess). Either way, I think it's more intuitive to use the flag once for each label pair.

Reply:
added a comment in the code to clarify

Reply:
Definitely, one flag at a time is what I would expect.